ui/lesson/ScriptingChallenge/library/secp256k1py.ts

```diff
@@ -65,7 +65,7 @@ class GE:
 				if E>>D&1:A+=F
 		return A
 	def __rmul__(A,a):
-		if A==G:return FAST_G.mul(a)
+		if A==G:return SECP256K1.FAST_G.mul(a)
```

Comment:
Not sure why this was necessary for this but hasn't cropped up in any of the other challenges.  I couldn't get the python to run without adding this.

Reply:
Oh is see, its called in the validation

Reply:
Should be fixed now